docs/reference/ml/trained-models/apis/update-trained-model-deployment.asciidoc

```diff
@@ -119,7 +119,8 @@ POST _ml/trained_models/elastic__distilbert-base-uncased-finetuned-conll03-engli
     "enabled": true,
     "min_number_of_allocations": 3,
     "max_number_of_allocations": 10
-  }
+  },
+  "num_threads": 1
```

Comment:
The update call shouldn't have `num_threads`. You set the number of threads upon creation and can never change it.